redash/models.py

```diff
@@ -411,7 +411,7 @@ def find_by_name(cls, org, group_names):
         return list(result)
 
     def __unicode__(self):
-        return unicode(self.id)
+        return text_type(self.id)
```

Comment:
This won't be enough, on Python 2 a `__unicode__` method is needed, on Python 3 it must be called `__str__`. A good explanation for this requirement can be found here: https://docs.djangoproject.com/en/1.11/topics/python3/#str-and-unicode-methods

Luckily six provides a nice decorator to cater to hybrid py2/3 code bases: [`@python_2_unicode_compatible`](https://pythonhosted.org/six/#six.python_2_unicode_compatible).

Simply decorate the models that need to be represented as a string, define a `__str__` method, and remove the `__unicode__` methods:

```python
@python_2_unicode_compatible
class Group(db.Model, BelongsToOrgMixin):
    # ...
    def __str__(self):
        return text_type(self.id)
```

Reply:
Nice to know...  Thanks.